.github/workflows/test.yml

```
@@ -29,7 +29,7 @@ jobs:
         run: |
           gem install awesome_bot
           awesome_bot \
-            -f README.md \
+            -f README.md,README-*.md \ 
```

Looking at the new CI checks, my suggested change doesn't appear to be working as I had intended. Looking at the tool itself (https://github.com/dkhamsing/awesome_bot), we can't mix and max the comma list and regex way to select files.

So let's try this.

```suggestion
            -f README*.md \ 
```